bettercli/cl/input.py

```
@@ -82,43 +85,45 @@ class Input:
             "cursor": Dict({
-                "fg-color": c.ANSIColors.cyan,
+                "fg-color": ANSIColors.cyan,
             }),
             "not cursor": Dict({
-                "fg-color": c.ANSIColors.reset,
+                "fg-color": ANSIColors.reset,
             }),
         })
     })
-    def __init__(self, prompt: str, suggestions: dict[str, str] = None, *, style: 'INPUT_STYLES' = {}):
+    def __init__(self, prompt: 'str', suggestions: 'list[str]' = None, *, style: 'INPUT_STYLES' = {}):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)